localdashboard_cumulus_std.bash

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
- Nit: consider .sh extension as other bash scripts in this repo are .bash
- Please consider moving to ./bin/
- Nit: add a brief description re: what this does
- Nit: consider adding this to either the readme or as an npm script to make it easier to find/awareness/etc.

Reply:
Also: this appears to be a script setup to automate building against a particular SIT env - is it a best practice to do this versus a development stack or the local test stack?